tmux/berkeley-from_bag.yml

```diff
@@ -20,12 +20,12 @@ root: <%= ENV["ANGEL_WORKSPACE_DIR"] %>
 # on_project_start: command
 on_project_start: |
   export ROS_NAMESPACE=/debug
-  export HL2_IP=${HL2_IP:-192.168.1.101}
+  export HL2_IP=${HL2_IP:-192.168.1.3}
```

Comment:
Take out local modifications like this IP change and I assume the commenting out below.